src/main/java/com/cloudbees/jenkins/support/SupportPlugin.java

```diff
@@ -396,6 +402,59 @@ public static void loadConfig() throws IOException {
             instance.load();
     }
 
+    private static final boolean logStartupPerformanceIssues = Boolean.getBoolean(SupportPlugin.class.getCanonicalName() + ".threadDumpStartup");
```

Comment:
Whatever you want, but if this causes issues, may need to turn it off separately to my mind